server/routes/api/auth.js

```diff
@@ -22,4 +25,52 @@ router.post('/login', (req, res) => {
     });
 });
 
+router.get('/logout', (req, res) => {
+  if (req.session.userId) {
+    req.session.destroy(err => {
+      err ? res.status(500).json(err) : res.status(200)
+    })
+  }
+  res.end()
+});
+
+router.post('/forgot', (req, res) => {
+  User.findOne(req.body)
```

Comment:
If password, first name, last name, etc is in the body, this will also have to match on those. While this route should only accept an email, it's misleading to not get a reset password token saved to the user when one of those other fields are submitted, and don't match as well. Because we're not providing any sort of validation for what the body should contain, let's make this easy and just extract the email. So we'll do a `.findOne()` with _just_ the email from the body.